bindings/python/client.py

```diff
@@ -132,15 +134,18 @@ def print_peer_info(console: ConsoleType, peers: List[Any]) -> None:
         elif p.flags & lt.peer_info.connecting:
             id = "connecting to peer"
         else:
-            id = p.client
+            try:
+                id = p.client.decode()
```

Comment:
`client` is always supposed to be a valid UTF-8 string. have you seen this not be a valid string?

Reply:
The conversion is just here because adding type stubs caused type checking to fail, so it needs to be converted.

I've used transmission a lot in the past, and using that, I've seen a lot of random data in the user agent field. I wasn't sure if libtorrent sanitizes the string, so I did a defensive conversion.